packages/tools/devtools/devtools-core/src/test/DefaultVisualizers.spec.ts

```diff
@@ -825,6 +825,222 @@ describe("DefaultVisualizers unit tests", () => {
 		expect(result).to.deep.equal(expected);
 	});
 
+	it("SharedTree: Leaf Handle", async () => {
+		const factory = SharedTree.getFactory();
+		const builder = new SchemaFactory("shared-tree-test");
+		const runtime = new MockFluidDataStoreRuntime();
+
+		const sharedTree = factory.create(
+			new MockFluidDataStoreRuntime({ idCompressor: createIdCompressor() }),
+			"test",
+		);
+
+		const sharedString = new SharedString(
+			runtime,
+			"test-string",
+			SharedString.getFactory().attributes,
+		);
+		sharedString.insertText(0, "Hello World!");
+
+		sharedTree.schematize(new TreeConfiguration(builder.handle, () => sharedString.handle));
+
+		const result = await visualizeSharedTree(
+			sharedTree as unknown as ISharedObject,
+			visualizeChildData,
+		);
+
+		const expected = {
```

Comment:
I'm a bit confused by the tests (unless they're not passing). The string value inside the SharedString is not part of the expected visual representation? The logic should process handles in a way that make the text visible somewhere here...?

Reply:
I haven't changed the `expected` test yet, as I wanted to first finish passing in the `typeMetaData`. Sorry for the confusion, `expected` for two of the SharedTree handle test suites should be fixed.